### ISSUE TITLE
Crossbeam manifesto

### ISSUE THREAD

**Issue by jeehoonkang** [crossbeam-rs/rfcs#7]
I just read @ticki's blog post that advocates hazard pointers (http://ticki.github.io/blog/fearless-concurrency-with-hazard-pointers/), and @stjepang's comment on it in reddit (https://www.reddit.com/r/rust/comments/6qz9p2/conc_an_efficient_concurrent_reclamation_system/dl20t02/).

Which makes me think that maybe it would be great to document why EBR, and when to use it.  In other words, I am suggesting an update on  [@aturon's blog post](https://aturon.github.io/blog/2015/08/27/epoch/), which will serve as the Crossbeam manifesto.  I hope we can produce such a document via an RFC process.

**Comment by ticki** [comment]
Yeah, definitely a good idea. My blog post isn't meant to "advocate for", because there are many times where ERB is fine. A how-to-choose guide would be interesting though.

**Comment by Firstyear** [comment]
Having both available inside of crossbeam would probably be a good option too, because then structures could be build using the choice of EBR or HP that suited correctly.
